packages/flutter/test/widgets/scrollable_test.dart

```diff
@@ -1427,6 +1427,33 @@ void main() {
     expect(syntheticScrollableNode!.hasFlag(ui.SemanticsFlag.hasImplicitScrolling), isTrue);
     handle.dispose();
   });
+
+  testWidgets('Scroll inertia cancel event', (WidgetTester tester) async {
+    // First try without cancelling.
```

Comment:
I guess we don't need the first try any more, since we no longer need the distance to compare with?